### PR TITLE
Add stack

### PR DESCRIPTION
**Pull request by OnderUlasKilic**
Add stack

### CODE REVIEW

File: lib/demos/stack_demo_view.dart [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_full_learn/core/random_image.dart';
+
+class StackDemoView extends StatelessWidget {
+  const StackDemoView({super.key});
+  final _cardHeight = 50.0;
```
Comment (BBarisKilic) [inline lib/demos/stack_demo_view.dart]: Add space between class constructor and variables.

```suggestion
  const StackDemoView({super.key});
  
  final _cardHeight = 50.0;
```

File: lib/demos/stack_demo_view.dart [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                  bottom: 0,
+                  width: _cardWith,
+                  child: const _CardCustom())
+            ],
+          ),
+        ),
+        const Spacer(
+          flex: 6,
+        )
+      ]),
```
Comment (BBarisKilic) [inline lib/demos/stack_demo_view.dart]: Add missing comma for readability.

```suggestion
      ],
      ),
```

File: lib/demos/stack_demo_view.dart [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            children: [
+              Positioned.fill(
+                bottom: _cardHeight / 2,
+                child: const RandomImage(),
+              ),
+              Positioned(
+                  height: _cardHeight,
+                  bottom: 0,
+                  width: _cardWith,
+                  child: const _CardCustom())
```
Comment (BBarisKilic) [inline lib/demos/stack_demo_view.dart]: Add missing comma for readability.

```suggestion
                  child: const _CardCustom(),
                  )
```

**Review by BBarisKilic — CHANGES_REQUESTED** [review]
Everything looks fine.

File: lib/101/stack_learn.dart [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            ),
+          ),
+          Positioned(
+            bottom: 0,
+            width: 25,
+            height: 100,
+            child: Container(
+              color: Colors.green,
+            ),
+          ),
```
Comment (BBarisKilic) [inline lib/101/stack_learn.dart]: Why these two child widgets are in the same position and are exactly the same, is there any reason? If not, please remove the last one.

**Review by BBarisKilic — CHANGES_REQUESTED** [review]

File: lib/demos/stack_demo_view.dart (hunk 1 above)
Comment (OnderUlasKilic) [reply lib/demos/stack_demo_view.dart]: Done.

File: lib/demos/stack_demo_view.dart (hunk 2 above)
Comment (OnderUlasKilic) [reply lib/demos/stack_demo_view.dart]: Done.

File: lib/demos/stack_demo_view.dart (hunk 3 above)
Comment (OnderUlasKilic) [reply lib/demos/stack_demo_view.dart]: Done.

File: lib/101/stack_learn.dart (hunk 4 above)
Comment (OnderUlasKilic) [reply lib/101/stack_learn.dart]: On off the code should be Positioned.fill will fix the code and update.

**Review by BBarisKilic — APPROVED** [review]

**Pull request merged**